dlib/cmake_utils/find_ffmpeg.cmake

```diff
@@ -1,29 +0,0 @@
-cmake_minimum_required(VERSION 3.8.0)
-
-message(STATUS "Searching for FFMPEG/LIBAV")
-find_package(PkgConfig)
-if (PkgConfig_FOUND)
-    pkg_check_modules(FFMPEG IMPORTED_TARGET
-        libavdevice
-        libavfilter
-        libavformat
-        libavcodec
-        libswresample
-        libswscale
-        libavutil
-    )
-    if (FFMPEG_FOUND)
-        message(STATUS "Found FFMPEG/LIBAV via pkg-config in `${FFMPEG_LIBRARY_DIRS}`")
-    else()
-        message(" *****************************************************************************")
-        message(" *** No FFMPEG/LIBAV libraries found.                                      ***")
-        message(" *** On Ubuntu you can install them by executing                           ***")
-        message(" ***    sudo apt install libavdevice-dev libavfilter-dev libavformat-dev   ***")
-        message(" ***    sudo apt install libavcodec-dev libswresample-dev libswscale-dev   ***")
-        message(" ***    sudo apt install libavutil-dev                                     ***")
-        message(" *****************************************************************************")
```

Comment:
These messages are really great, we should totally keep printing them.  Otherwise it's hard to know what to do and these make it super easy.